src/components/PassthroughForm.vue

```diff
@@ -0,0 +1,126 @@
+<template>
```

Comment:
Hmm yeah I think it would be a good idea to update this component a bit and move it to the lib... I'm sure other people will also want to be able to slot the uglier django forms into their nicer frontend to make it look better.